Tests/AztecTests/Converters/AttributesToStringAttributes/BoldElementAttributeConverterTests.swift

```diff
@@ -1,5 +1,8 @@
 import XCTest
 @testable import Aztec
+#if SWIFT_PACKAGE
+@testable import HTMLParser
+#endif
```

Comment:
I'm unsure why this `@testable import` has to be gated by `#if SWIFT_PACKAGE` (while other imports don't)? 🤔

Reply:
It has to do with the difference between how CocoaPods and SwiftPM build the library, or rather libraries.

In SwiftPM, we now have the standalone HTMLParser library and Aztec depends on it.

But in CocoaPods, I didn't expose HTMLParser. That's a bit of an inconsistency, I'll admit. But it seemed fine to only work in the context of SwiftPM, between the state of CocoaPods and the fact that this change was just to unblock SwiftPM in Woo.

But this situation resulted in a few build differences. The HTMLParser sources are part of Aztec when building with CocoaPods, but when building for SwiftPM (the default in Xcode and xcodebuild/CLI) they aren't.

Reply:
Ok I think I see (even though this is a bit much for my mushy Friday evening brain to take on fully 😅)

This seems like an important explanation, and something someone other that you (who worked on this migration directly and are thus familiar with the kind of hybrid/unbalanced setup you had to end up with) would require some digging to understand this subtlety and the reason behind it.

As such, I'll suggest to document it somewhere in the code. Like maybe a code comment explaining all that in the key/main/root place in the codebase where this `#if` case happens for the first time, then code comments next to all other `#if` you had to similarly add just pointing to that longer comment/explanation (to avoid copy/pasting same long explanatory comment at every single place)? wdyt?